test/hotspot/jtreg/compiler/igvn/ExpressionFuzzer.java

```diff
@@ -332,6 +338,16 @@ public static String generate(CompileFramework comp) {
             }
         }
 
+        // Generate expressions with any scalar numeric types.
+        for (CodeGenerationDataNameType type : SCALAR_NUMERIC_TYPES) {
+            for (int i = 0; i < 2; i++) {
```

Comment:
Only a question: what does this loop do? And why do we have only 2 iterations here, but 10 for `PRIMITIVE_TYPES`?

Reply:
It `Generate[s] expressions with any scalar numeric types.` ;)
The question is just how many per (output) type. Here we do 2 for each type.

The 2 vs 10 is quite arbitrary. I did not want to increase the runtime of the test too much. For now, focusing more on the primitive types an operations is probably good, Float16 is still rather niche. But we can change the balance in the future.